### PR TITLE
still preparing for polar in enterprise

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
Less is more

### CODE REVIEW

File: taipy/gui/data/array_dict_data_accessor.py [hunk 1]
```diff
@@ -26,8 +25,8 @@ class _ArrayDictDataAccessor(_PandasDataAccessor):
     def get_supported_classes() -> t.List[str]:
         return [t.__name__ for t in _ArrayDictDataAccessor.__types]  # type: ignore
 
-    def _get_dataframe(self, value: t.Any) -> t.Union[t.List[pd.DataFrame], pd.DataFrame]:
-        if isinstance(value, list):
+    def to_pandas(self, value: t.Any) -> t.Union[t.List[pd.DataFrame], pd.DataFrame]:
+        if isinstance(value, (list, tuple)):
```
Comment (FabienLelaquais) [inline taipy/gui/data/array_dict_data_accessor.py:29]: `isinstance(value, Iterable)`? I'm thinking numpy arrays...

File: taipy/gui/data/data_accessor.py [hunk 2]
```diff
@@ -17,18 +17,24 @@
...
+
     @staticmethod
     @abstractmethod
     def get_supported_classes() -> t.List[str]:
         pass
 
     @abstractmethod
     def get_data(
-        self, guiApp: t.Any, var_name: str, value: t.Any, payload: t.Dict[str, t.Any], data_format: _DataFormat
+        self, var_name: str, value: t.Any, payload: t.Dict[str, t.Any], data_format: _DataFormat
```
Comment (FabienLelaquais) [inline taipy/gui/data/data_accessor.py:37]: 😁

**Review by FabienLelaquais — APPROVED** [review]
🤯

**Review by dinhlongviolin1 — APPROVED** [review]
Lgtm!

File: taipy/gui/data/array_dict_data_accessor.py (hunk 1 above)
Comment (FredLL-Avaiga) [reply taipy/gui/data/array_dict_data_accessor.py:29]: numpy are dealt with in numpy_data_accessor :-)

**Pull request merged**